cmd/nightfalldlp/main.go

```diff
@@ -17,6 +17,7 @@ const (
 	nightfallConfigFileName = ".nightfalldlp/config.json"
 	githubActionsEnvVar     = "GITHUB_ACTIONS"
 	githubTokenEnvVar       = "GITHUB_TOKEN"
+	githubBaseUrlEnvVar     = "BASE_URL"
```

Comment:
TODO: update readme docs on https://github.com/nightfallai/nightfall_dlp_action and https://github.com/nightfallai/nightfall_circle_orb on how to use this